marimbabot_planning/src/lilypond_to_hitpoint.cpp

```diff
@@ -106,87 +107,111 @@ std::vector<std::tuple<geometry_msgs::PoseStamped, double, double>> lilypond_to_
 }
 
 
-int main(int argc, char **argv)
+/**
+ * @brief ROS node that subscribes to the recognized lilypond sequence and publishes a marker at the correct time
+ * class is needed to access the publisher inside the callback function
+*/
+class LilypondMotionConverter
 {
-    ros::init(argc, argv, "marimba_move_lilypond_input_viz");
-    ros::NodeHandle node_handle;
-    ros::AsyncSpinner spinner(1);
```

Comment:
would be nice to keep AsyncSpinner for threads

Reply:
done